pkg/openwsn/doc.txt

```diff
@@ -0,0 +1,217 @@
+/**
+ * @defgroup pkg_openwsn   OpenWSN network stack
+ * @ingroup  pkg
+ * @ingroup  net
+ * @brief    Provides a RIOT adaption of the OpenWSN network stack
+ * @see      https://github.com/openwsn-berkeley/openwsn-fw
+ *
+ *
+ * # OpenWSN RIOT Port
+ *
+ * This implementation integrates the [OpenWSN](https://github.com/openwsn-berkeley/openwsn-fw)
+ * full stack (UDP, IPv6, RPL, 6TiSCH) into RIOT.
+ *
+ * We provide a new RIOT "board" to the OpenWSN software. In this way we connect
+ * RIOT's hardware abstraction to OpenWSN's interfaces. Furthermore, we make use
+ * of the simple scheduling mechanism in OpenWSN which we run in a RIOT thread
+ * with second highest priority after the radio thread (THREAD_PRIORITY_MAIN - 4).
+ *
+ * The current PORT of OpenWSN currently needs a root node that works along an
+ * external tool that performs routing and handles join procedure:
+ * [Openvisualizer](https://github.com/openwsn-berkeley/openvisualizer)
+ *
+ * ## Joining a network
+ *
+ * The first thing a new mote will need to do is to find a network. On boot it
+ * will actively be listening for enhanced beacons. Once a beacon is received it
+ * will adjust its timers drift and synchronize with the network. Re-synchronization
+ * will be happening constantly to compensate for oscillator and timer drifts.
+ *
+ * Once synchronized the node will need to join the network. OpenWSN uses CoJP
+ * [constrained join protocol](https://datatracker.ietf.org/doc/draft-ietf-6tisch-minimal-security/)
+ * OpenWSN stack only handles Join Requests. The JRC is not running on the root
```

Comment:
JRC ? (Join Request Controller ?). Maybe give explain the acronyms, here and below (DIS, DIO, SRH, etc)